packages/react/src/internal/selectedItems/SelectedItems.tsx

```diff
@@ -239,6 +239,10 @@ export const SelectedItems = <OptionType,>({
                 onKeyDown: (event) => {
                   // some browsers navigate back when Backspace is pressed
                   if (event.key === 'Backspace') event.preventDefault();
```

Comment:
Should we fire onRemove also when user removes selected item with backspace keypress?

Reply:
IIRC, Downshift does it by default, so we do not need to handle it.